src/App.js

```diff
@@ -9,6 +9,23 @@ export default function App() {
   const [activities, setActivities] = useLocalStorageState("activities", {
     defaultValue: [],
   });
+  const isGoodWeather = true;
+  const isBadWeather = false;
+
+  function filteredGoodWeatherActivities(isGoodWeather) {
+    return activities.filter((activity) => {
+      if (activity.forNiceWeather === isGoodWeather) {
+        return true;
+      } else {
+        return false;
+      }
+    });
+  }
+
+  const filteredBadWeatherActivities = activities.filter(
+    (activity) => activity.forNiceWeather === isBadWeather
+  );
```

Comment:
this works for now and gives you the chance to return a list with good weather and bad weather activities each, but after having switched to the API fetch, you should only need one variable for the filtered activities and one list returning those activities